torch/script.h

```diff
@@ -5,6 +5,7 @@
 #include <torch/csrc/autograd/custom_function.h>
 #include <torch/csrc/autograd/generated/variable_factories.h>
 #include <torch/csrc/autograd/grad_mode.h>
+#include <torch/csrc/jit/python/utf8_decoding_ignore.h>
```

Comment:
nit: is this necessary?

Reply:
This is needed to for registering bindings in the follow-up change in [torchtext/csrc/register_torchbindings.cpp](https://github.com/pytorch/text/pull/2126/files#diff-2efc883b84a0f9a8a4abbe2e3da54037ff7d8e6bc4403fd2d7aefde5afa7f044), since it only includes torch/script.h.

Reply:
hmm can we just include `torch/csrc/jit/python/utf8_decoding_ignore.h` in register_torchbindings.cpp?

not a big deal but since this is sort of a private api, we probably don't need to include it in torch/script.h (as you can see the current script.h includes a pretty minimal list of .h files)

Reply:
Unfortunately, it fails with 
```
fbcode/pytorch/text/torchtext/csrc/register_torchbindings.cpp:215:20: error: no member named 'setUTF8DecodingIgnore' in namespace 'torch::jit'
      &torch::jit::setUTF8DecodingIgnore);
 ```
 when `utf8_decoding_ignore.h` is moved from torch/script.h to torchtext/csrc/register_torchbindings.cpp.

Reply:
hmm and this is after adding the .h include to both torchtext/csrc/register_pybindings.cpp and torchtext/csrc/register_torchingbindings.cpp

Reply:
right